trace-utils/src/trace_utils.rs

```diff
@@ -183,23 +206,107 @@ pub async fn send(data: Vec<u8>) -> anyhow::Result<()> {
     let client: Client<_, hyper::Body> = Client::builder().build(https);
     match client.request(req).await {
         Ok(_) => {
-            println!("Successfully sent traces");
+            info!("Successfully sent traces");
         }
-        Err(e) => println!("Failed to send traces: {}", e),
+        Err(e) => error!("Failed to send traces: {}", e),
     }
     Ok(())
 }
 
-#[cfg(test)]
-mod tests {
-    use std::collections::HashMap;
+pub fn get_root_span_index(trace: &Vec<pb::Span>) -> anyhow::Result<usize> {
```

Comment:
I think you can just use Result from std here

Reply:
Lots of code in libdatadog uses `anyhow::Result` specifically, either for its convenience helpers like `anyhow::bail!` (like this code does), or for its `.context` helpers for chaining errors together. I think it's fine to keep it this way for now, although last time this sort of thing came up there was another error library mentioned that may be better. Maybe we should investigate that a bit more, but for now it's fine, I think.